sdk/keyvault/azure-keyvault-nspkg/MANIFEST.in

```diff
@@ -0,0 +1,3 @@
+include *.md
+include azure/__init__.py
+include azure/keyvault/__init__.py
```

Comment:
Do any modifications need to be made to the MANIFEST files for keys/secrets/certificates?